lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -18,10 +18,10 @@
 require 'time'
 require 'yaml'
 require 'google/apis'
-require 'google/apis/logging_v1beta3'
-require 'google/logging/v1/logging_pb'
```

Comment:
You should be able to completely remove `lib/google` on this branch, since we're now using the gRPC bindings from `google-cloud-logging`...